docs/data/changelog/ChangeLog.js

```diff
@@ -188,102 +194,161 @@ const blogs = [
+            fontWeight="bold"
+            mb={0.2}
+          >
             {props.blog.title}
           </Typography>
-        </Box>
-
-        <Box sx={{ p: 1 }}>
-          <Typography variant="body2" component="div" color="text.secondary">
+          <Typography component="div" color="text.secondary" variant="body2">
             {props.blog.description}
           </Typography>
-
-          <List sx={{ pb: 0, mt: 0.5 }}>
-            {props.blog.highlightList.map((item) => (
-              <ListItem
-                key={item.title}
-                sx={{
-                  py: 0.5,
-                  px: 0.5,
-                  display: 'flex',
-                  alignItems: 'flex-start',
-                  lineHeight: '22px',
-                  [`&:before`]: {
-                    content: '"•"',
-                    color: 'grey',
-                    lineHeight: '22px',
-                  },
-                }}
-              >
-                <Link
-                  href={item.url}
-                  sx={{ pl: 1, fontWeight: 500 }}
-                  variant="body2"
-                >
-                  {item.title}
-                </Link>
-              </ListItem>
-            ))}
-          </List>
-        </Box>
-      </CardContent>
-      <CardActions
-        sx={{
-          justifyContent: 'center',
-          flex: '1 1 auto',
-          m: 0,
-          alignItems: 'end',
-        }}
-      >
-        <Button sx={{ width: '100%' }} href={props.blog.url}>
+        </div>
+        <Button
+          component="a"
+          size="small"
+          variant="outlined"
+          href={props.blog.url}
+          sx={{ height: 'fit-content', flexShrink: 0 }}
+        >
           Read more
         </Button>
-      </CardActions>
+      </Box>
+      <Divider />
+      <List sx={{ p: 2, pt: 1.5 }}>
+        {props.blog.highlightList.map((item) => (
+          <ListItem
+            key={item.title}
+            sx={{
+              py: 0.5,
+              px: 0.5,
+              display: 'flex',
+              alignItems: 'flex-start',
+              lineHeight: '22px',
+              [`&:before`]: {
+                content: '"➞"',
+                opacity: '50%',
+                color: 'primary.300',
+                lineHeight: '22px',
+              },
+            }}
+          >
+            <Link
+              href={item.url}
+              variant="body2"
+              sx={{ pl: 1.5, fontWeight: 'medium' }}
+            >
+              {item.title}
+            </Link>
+          </ListItem>
+        ))}
+      </List>
     </Card>
   );
 }
 
-export default function WhatsNewLayoutNoSnap() {
+export default function ChangeLog() {
   return (
-    <Grid container spacing={2} sx={{ pt: 2, pb: 4 }}>
-      {blogs.map((blog) => (
-        <Grid item xs={12} sm={6} md={4} sx={{ flexGrow: 1 }} key={blog.title}>
-          <BlogCard blog={blog} />
-        </Grid>
-      ))}
-    </Grid>
+    <BrandingCssVarsProvider>
```

Comment:
I added this component here as a way for this demo to access the `brandingTheme` customizations. Otherwise, it'd pick up from the default Material UI (Design) theme. But it seems this is a bit buggy when switching between light & dark mode... I'd appreciate some help here as to how to do it in a better way!

Reply:
About this one I can provide explainantion, but did not manage to fix it for now.


1. You need to add `<BrandingCssVarsProvider>` because all the demo are wrapped into the default MUI ThemeProvider to create demos with the style developpers will get in their app and not our custom docs theme.

2. For the `applyDarkStyles` type, let's say that type definition are not always nicely supported between X and Core. Might just be TS that does not find the type definition. Otherwise, you can just redefine this function:

```jsx
function applyDarkStyles(obj: object) {
  return {
    ':where([data-mui-color-scheme="dark"]) &': obj,
  };
}
```

3. The previouse point shows that branding theme relies on the `data-mui-color-scheme` which is not weel supported by X docs. Here is a recoding of the code home page, and some x docs page. As you can see the dark/ligh swich does not update the `data-mui-color-scheme` in the `<html>` element 😱

https://github.com/mui/mui-x/assets/45398769/74d8a73a-83ae-4a8a-96c3-2d5a09d37f09


https://github.com/mui/mui-x/assets/45398769/b2c49133-03bd-476c-9333-c3aa52bf9440

Reply:
Any thoughts on what to do, precisely? 😬 I wasn't sure how to insert that function snippet you shared!